service/policy/db/migrations/20240806142109_add_resource_mapping_groups.sql

```diff
@@ -0,0 +1,29 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS resource_mapping_groups (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    namespace_id UUID NOT NULL REFERENCES attribute_namespaces(id),
```

Comment:
```suggestion
    namespace_id UUID NOT NULL REFERENCES attribute_namespaces(id) ON CASCADE DELETE,
```

Reply:
if the namespace is deleted, we want to delete these groups

Reply:
Good catch, and it should be `ON DELETE CASCADE`